### PR TITLE
BS-153 Update cache and log configuration

### PR DESCRIPTION
**Pull request by p-tombez**

### CODE REVIEW

File: 9.0/templates/nginx.conf.tmpl [hunk 1]
```diff
@@ -61,36 +86,54 @@ http {
...
+
     location /longpolling/im/poll { proxy_pass http://{{ $odoo_host }}:8072; }
     location /longpolling/poll { proxy_pass http://{{ $odoo_host }}:8072; }
 
     location ~* /[^/]+/static/ {
-        proxy_cache_valid 200 60m;
-        proxy_buffering on;
-        expires 864000;
-        proxy_pass http://{{ $odoo_host }}:8069;
+      try_files @cached;
```
Comment (guewen) [inline 9.0/templates/nginx.conf.tmpl]: Is `try_files` useful without fallback?

Comment (p-tombez) [reply 9.0/templates/nginx.conf.tmpl]: @guewen I think it's the only way to call for a named location without touching the initial request

Comment (guewen) [reply 9.0/templates/nginx.conf.tmpl]: ok!

**Comment by guewen** [comment]
@p-tombez I have this error with the latest image now (11.0-latest):

```
nginx_1  | 2018-01-17T10:26:59Z 07a5d7856ae3 /usr/local/bin/confd[7]: INFO Backend set to env
nginx_1  | 2018-01-17T10:26:59Z 07a5d7856ae3 /usr/local/bin/confd[7]: INFO Starting confd
nginx_1  | 2018-01-17T10:26:59Z 07a5d7856ae3 /usr/local/bin/confd[7]: INFO Backend nodes set to
nginx_1  | 2018-01-17T10:26:59Z 07a5d7856ae3 /usr/local/bin/confd[7]: INFO /etc/nginx/nginx.conf has md5sum f7984934bd6cab883e1f33d5129834bb should be fe248e04a5e4eef11ff988e4201dede9
nginx_1  | 2018-01-17T10:26:59Z 07a5d7856ae3 /usr/local/bin/confd[7]: INFO Target config /etc/nginx/nginx.conf out of sync
nginx_1  | 2018-01-17T10:26:59Z 07a5d7856ae3 /usr/local/bin/confd[7]: INFO Target config /etc/nginx/nginx.conf has been updated
nginx_1  | 2018/01/17 10:26:59 [emerg] 1#1: invalid number of arguments in "try_files" directive in /etc/nginx/nginx.conf:113
nginx_1  | nginx: [emerg] invalid number of arguments in "try_files" directive in /etc/nginx/nginx.conf:113
```
Could you check?

**Comment by p-tombez** [comment]
@guewen I just saw that a named location doesn't count as a path in ```try_files```. I'll add a fallback as it is needed.

**Pull request merged**